scripts/test-triton.sh

```diff
@@ -197,6 +197,12 @@ run_core_tests() {
   # run test_line_info.py separately with TRITON_DISABLE_LINE_INFO=0
   TRITON_DISABLE_LINE_INFO=0 TRITON_TEST_SUITE=line_info \
     pytest -k "not test_line_info_interpreter" --verbose --device xpu language/test_line_info.py
+
+  TRITON_DISABLE_LINE_INFO=1 TRITON_TEST_SUITE=tools \
+    pytest --verbose --device xpu tools/test_disasm.py
+
+  TRITON_DISABLE_LINE_INFO=1 TRITON_TEST_SUITE=tools \
+    pytest --verbose --device xpu tools/test_aot.py
```

Comment:
`TRITON_TEST_SUITE` needs to be unique, otherwise report from the first run will be overwritten by the second one.
```suggestion
  TRITON_DISABLE_LINE_INFO=1 TRITON_TEST_SUITE=tools \
    pytest --verbose --device xpu tools/test_disasm.py tools/test_aot.py
```